scenes_test.go

```diff
@@ -84,14 +84,20 @@ func runScenes(t *testing.T, name string, options map[string]interface{}) error
 func runPlanner(fileName string, options map[string]interface{}) error {
 	start := time.Now()
 
+	// Check the frame we are planning against, in case we have brought in a static frame for an attached end effector
+	evalFrame := testArmFrame
+	if scene.EndEffectorFrame != nil {
```

Comment:
`scene` here is actually `config`.
[opt] : use same name to avoid confusion about where `scene` came from.